README.md

```diff
@@ -106,6 +106,8 @@ The most basic tag type is a simple variable. A `{{name}}` tag renders the value
 
 All variables are HTML-escaped by default. If you want to render unescaped HTML, use the triple mustache: `{{{name}}}`. You can also use `&` to unescape a variable.
 
+To print `{{name}}` as a string literal, the default delimiters must be changed. See the ["Set Delimiter'](https://github.com/janl/mustache.js#set-delimiter) section for more information about custom delimiters.
+
```

Comment:
I wouldn't use the phrase "string literal". A string literal is literally `"something like this"`, which isn't exactly what this means here. I'd say something like:

> If you want to mustache tags — like `{{name}}` — literally in your template, the default delimiters must be changed. See …